tests/pc_parser.cpp

```diff
@@ -0,0 +1,40 @@
+#include "cps/pc_compat/pc_loader.hpp"
+#include <fstream>
+#include <gtest/gtest.h>
+
+namespace cps::utils::test {
+    namespace {
```

Comment:
Let's add some "negative tests", too. Files that aren't valid pc files are important to test to ensure we give back helpful error messages instead of crashing or something.

Reply:
My current goal, at least for this PR, is for this to consume correctly formatted PC files. If someone is troubleshooting their handwritten pc file, probably cps-config is not their best tool. Maybe we can follow up with a PR that turns on location tracking and give more readable error messages, but I think for this PR, let's first work with known-good pc files, which I believe is by far the majority use case.